api/pkg/apps/webapp/templates/team.gohtml

```diff
@@ -43,6 +55,152 @@
         </div>
     </div>
 
+    <script>
+        let {Subject, BehaviorSubject, pipe, Observable} = rxjs;
+        let {map, debounceTime, switchMap} = rxjs.operators;
+        let {ajax} = rxjs.ajax;
+
+        // Handles the search box for the parties dropdown
+        let partySearchSubject = new Subject()
+        let partySearch$ = partySearchSubject.asObservable().pipe(
+            debounceTime(200),
```

Comment:
you can add 

a `map(s => s.trim())` to remove whitespaces on the search query
a `distinctUntilChanged()` (to not re-run the same query over) 
a `filter(s => s.length > 5)`  (to wait until the input is at least 5 chars long

Reply:
I think we should omit `filter(s => s.length > 5)` since it would prevent searching for people with shorter first names. Added the other two.